src/RequestLog/Middleware/LogRequest.php

```diff
@@ -66,28 +102,17 @@ public function terminate($request, $response): void
                 $executionTime = 0;
             }
 
-            // We need to check if the path of the current request is blacklisted
-            // if it is we out-out here and to not write a RequestLog entry
-            if ($this->routeIsBlacklisted($request)) {
-                return;
-            }
-
-            RequestLog::create([
-                'client_ip'          => $request->ip(),
-                'user_agent'         => $request->userAgent(),
-                'method'             => $request->method(),
-                'status'             => $response->status(),
-                'url'                => $request->url(),
-                'root'               => $request->root(),
-                'path'               => $request->path(),
-                'query_string'       => SecurityUtility::getQueryWithMaskingApplied($request),
-                'request_headers'    => SecurityUtility::getHeadersWithMaskingApplied($request),
-                'request_body'       => SecurityUtility::getBodyWithMaskingApplied($request) ?: '{}',
+            // Update the receivedRequest with response data, then save changes to the database.
+            $this->receivedRequest->update([
```

Comment:
Hvis du gerne vil bruge save() til sidst, så brug filll() her i stedet for update(). Alternativt fjern save() nedenunder :)